packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -3,16 +3,20 @@
 
 import * as React from "react";
 import ReactDOM from "react-dom";
+import {Popper} from 'react-popper';
 import {StyleSheet} from "aphrodite";
 
 import Color, {fade} from "@khanacademy/wonder-blocks-color";
+import {maybeGetPortalMountedModalHostElement} from "@khanacademy/wonder-blocks-modal";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {View} from "@khanacademy/wonder-blocks-core";
 
 import typeof ActionItem from "./action-item.js";
 import typeof SelectItem from "./select-item.js";
 import typeof SeparatorItem from "./separator-item.js";
 
+import visibilityModifierDefaultConfig from "../../wonder-blocks-tooltip/util/visibility-modifier.js";
```

Comment:
🛑 Oof, yeah, this isn't great. It feels like something that we should probably copy over or put in a shared location. I think for now copying it over and leaving comments on both of them saying we should find a way to re-use them makes sense.

Reply:
Agreed. If this gets used outside of tooltip we should consider having a `wonder-blocks-popper` package or consider putting this stuff into the `wonder-blocks-layout` package to share it